src/libtriton/arch/register.cpp

```diff
@@ -26,6 +26,14 @@ namespace triton {
     }
 
 
+    RegisterSpec::RegisterSpec(const RegisterSpec& other)
+      : BitsVector(other) {
+        this->name    = other.name;
+        this->id      = other.id;
+        this->parent  = other.parent;
+    }
```

Comment:
You should not define this function as it is already generated by the compiler and here you only give a worse possible version (the compiler can do better), you disable the move constructor and add more code to maintains.